@codex please fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
 ✓ tests/eslintPluginPresence.test.ts (1 test) 6ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 7ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 6ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 4ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 6ms
 ✓ tests/prismSvelte.test.ts (1 test) 15ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 4ms
 ✓ tests/progressionBalance.test.ts (1 test) 43ms
 ✓ tests/domException.test.ts (1 test) 5ms
 ✓ tests/indexeddb.server.test.ts (1 test) 225ms
 ✓ frontend/tests/sanity.test.ts (1 test) 3ms
[warn] src/pages/processes/__tests__/Processes.spec.ts
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 18205ms
   × Prettier formatting > frontend sources are formatted 18203ms
     → Command failed: npm run format:check

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5
      6|   test('frontend sources are formatted', () => {
      7|     const frontendDir = path.join(__dirname, '../../frontend');
      8|     execSync('npm run format:check', {
       |     ^
      9|       cwd: frontendDir,
     10|       stdio: 'inherit',

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 287 passed (288)
      Tests  1 failed | 996 passed (997)
   Start at  08:08:20
   Duration  157.37s (transform 9.11s, setup 10.34s, collect 44.68s, tests 100.80s, environment 179.29s, prepare 33.48s)


Error: Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: { status: 1, signal: null, output: [ null, null, null ], pid: 9343, stdout: null, stderr: null }

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/e2e/remote-release-smoke.spec.ts
+++ b/frontend/e2e/remote-release-smoke.spec.ts
@@ -128,24 +128,20 @@ async function runProcessLifecycle(page: Page): Promise<void> {
     await page.goto('/processes');
     await waitForHydration(page);
 
-    const processRow = page
-        .locator('tr, .process-row, [data-testid="process-row"], li, article, section, div')
-        .filter({ has: page.locator('button').filter({ hasText: /start/i }) })
-        .first();
-    const startButton = processRow.locator('button').filter({ hasText: /start/i }).first();
+    const detailLink = page.getByRole('link', { name: /view details/i }).first();
+    await expect(detailLink, 'Expected at least one process details link').toBeVisible();
+    await detailLink.click();
+    await expect(page).toHaveURL(/\/processes\/(?!manage$|create$)[^/]+$/);
+
+    const processContainer = page.locator('.process-view, main').first();
+    const startButton = processContainer.getByRole('button', { name: /start/i }).first();
 
     await expect(startButton, 'Expected at least one process start button').toBeVisible();
     await startButton.click();
 
-    const cancelButton = processRow
-        .locator('button')
-        .filter({ hasText: /cancel/i })
-        .first();
+    const cancelButton = processContainer.getByRole('button', { name: /cancel/i }).first();
 
-    const collectButton = processRow
-        .locator('button')
-        .filter({ hasText: /collect/i })
-        .first();
+    const collectButton = processContainer.getByRole('button', { name: /collect/i }).first();
 
     if (await cancelButton.isVisible({ timeout: 5_000 }).catch(() => false)) {
         await cancelButton.click();
@@ -159,7 +155,7 @@ async function runProcessLifecycle(page: Page): Promise<void> {
         return;
     }
 
-    const activeMarker = processRow.locator(
+    const activeMarker = processContainer.locator(
         '[data-status="active"], .process-running, :text("In progress")'
     );
     const state = await Promise.race([

--- a/frontend/src/pages/processes/__tests__/Processes.spec.ts
+++ b/frontend/src/pages/processes/__tests__/Processes.spec.ts
@@ -64,9 +64,9 @@ describe('Processes list route contract', () => {
         customListMock.mockResolvedValue([]);
         render(Processes, { props: { builtInProcesses } });
 
-        expect(screen.getByRole('link', { name: 'Create a new process' }).getAttribute('href')).toBe(
-            '/processes/create'
-        );
+        expect(
+            screen.getByRole('link', { name: 'Create a new process' }).getAttribute('href')
+        ).toBe('/processes/create');
         expect(screen.getByRole('link', { name: 'Manage processes' }).getAttribute('href')).toBe(
             '/processes/manage'
         );

